### PR TITLE
Refactor FE code sample

### PR DESCRIPTION
**Pull request by reggie7**
Refactors from [feature/inlineedit](https://github.com/headwirecom/peregrine-cms/tree/feature/inlineedit) moved on top of `issues/93`.

### CODE REVIEW

File: admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/toolingpage.html [hunk 1]
```diff
@@ -112,61 +112,71 @@
...
+        nodes: {}
+    }
+
     // switch between old and new implementation
     var newImpl = true
-    var $perAdminView = { state: {}, adminPage: {}, adminPageStaged: {}, admin: {}, pageView: { view: 'desktop'}, nodes: {} }
-
-    if(newImpl) {
+    if (newImpl) {
+        var loadComponent = $perAdminApp.loadComponent;
```
Comment (reusr1) [inline admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/toolingpage.html]: not sure I like this one - a var in javascript is global and const are not supported in all older browsers

consider this script:
```
<script>
	if(true) {
		var test = 'am I surviving my block?';
	}
</script>

<script>
	console.log(test)
</script>
```

the output will be

```am I surviving my block?```

hence this change may have side effects

Comment (reggie7) [reply admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/toolingpage.html]: Changed, please check.

File: admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/renderer.html [hunk 2]
```diff
@@ -38,14 +38,10 @@
...
-    Vue.component('vue-multiselect', VueMultiselect.default)
+    Vue.component('vue-form-generator', window.VueFormGenerator.component);
+    Vue.component('vue-multiselect', window.VueMultiselect.default);
     Vue.component('datetime', window.VueDatetime.Datetime);
-    Vue.component('Trumbowyg', VueTrumbowyg.default);
-    Vue.directive('images-loaded', VueImagesLoaded)
-    Vue.use(VueFormWizard)
+    Vue.component('Trumbowyg', window.VueTrumbowyg.default);
+    Vue.directive('images-loaded', window.VueImagesLoaded);
+    Vue.use(VueFormWizard);
```
Comment (flx-sta) [inline admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/renderer.html:46]: why exactly is it `window.XYZ` everywhere except for `VueFormWizard`
I'd prefer NOT to have everything inside the window. That is called "window-object-pollution" and is a bad thing to do

**Comment by flx-sta** [comment]
@reggie7 how exactly do i test the functionality ?

File: admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/renderer.html (hunk 2 above)
Comment (reggie7) [reply admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/renderer.html:46]: Simple omission, fixed.

Comment (reggie7) [reply admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/toolingpage/renderer.html:46]: > @reggie7 how exactly do i test the functionality ?

This is a simple cleanup/refactor to allow better readability of code. For checking it's enough to smoke-test a couple of pages to confirm their editing is still possible as before.

**Review by flx-sta — APPROVED** [review]

**Comment by reusr1** [comment]
we should reevaluate this PR and see what parts of it still make sense and maybe break it up into smaller junks

**Comment by reggie7** [comment]
@reusr1 I've created #668 instead. Some changes don't make sense anymore due to conflicts, but the remainder is small enpugh for a single PR I think.

**Pull request closed without merging**